Donation-platform/src/main/java/com/google/servlets/TrendingQueryServlet.java

```diff
@@ -29,28 +33,29 @@
 import com.google.appengine.api.datastore.PreparedQuery;
 import com.google.appengine.api.datastore.Query;
 import com.google.appengine.api.datastore.Query.SortDirection;
-import com.google.sps.data.Comment;
 import com.google.appengine.api.users.UserService;
 import com.google.appengine.api.users.UserServiceFactory;
 
 /**Servlet that handles requests for the trending page**/
 @WebServlet("/trending-query")
 
-public class TrendingQueryServlet extends HTTPServlet {
-    @Override
+public class TrendingQueryServlet extends HttpServlet {
+
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
         Gson gson = new Gson();
 
         // Find the trending charities.
         FindTrendingCharities findTrending = new FindTrendingCharities();
-        Collection<Charity> answer =
-            findTrending.query();
+        Collection<Charity> answer = findTrending.query();
+        System.out.println(answer);
 
         // Convert the collection of charities to JSON
         String jsonResponse = gson.toJson(answer);
+        System.out.println(jsonResponse);
 
         // Send the JSON back as the response
         response.setContentType("application/json");
         response.getWriter().println(jsonResponse);
```

Comment:
Use assert statements for comparing and testing and not print statements.

Reply:
yes, sorry that was left over from when I was trying to get the servlet properly connected to the js and html.